src/MenuItem.js

```diff
@@ -96,12 +106,26 @@ const MenuItem = ({
     )
 }
 
+/**
+ * @typedef {Object} PropTypes
+ * @static
+ *
+ * @prop {string|Node} label
+ * @prop {string} [value]
+ * @prop {string} [href]
+ * @prop {function} [onClick] - Click handler called with `value` as the sole argument
+ * @prop {string} [className]
+ * @prop {Element} [children]
+ * @prop {Element} [icon]
+ * @prop {boolean} [dense]
+ * @prop {boolean} [active]
+ * @prop {boolean} [disabled]
+ */
 MenuItem.propTypes = {
-    label: propTypes.oneOfType([propTypes.string, propTypes.object]).isRequired,
+    label: propTypes.oneOfType([propTypes.string, propTypes.node]).isRequired,
 
-    value: propTypes.any,
+    value: propTypes.string,
```

Comment:
Possible breaking change: this prop has become more strict.

Reply:
agreed